rules/falco_rules.yaml

```diff
@@ -74,6 +74,11 @@
 - macro: modify
   condition: (rename or remove)
 
+# %evt.arg.flags available for evt.dir=>, but only for umount2
+# %evt.arg.name is path and available for evt.dir=<
+- macro: umount
```

Comment:
is this for an upcoming rule or for addressing https://github.com/falcosecurity/falco/issues/2443#issuecomment-1471289452? in that case, do we want macros also for the event types (e.g pipe,pipe2) already put in place?